apps/braze/src/utils.ts

```diff
@@ -61,15 +61,15 @@ export function removeHypens(str: string) {
 
 export async function updateConfig(
```

Comment:
💭 Nitpick: I think it could make sense to move this to updateContentBlock, even if the dialog is using it